src/stravalib/client.py

```diff
@@ -1004,10 +1007,26 @@ def get_activity_zones(
         :class:`list`
             A list of :class:`stravalib.model.BaseActivityZone` objects.
 
+        Notes
+        -----
+        Activity zones require a Strava premium account.
+        The Strava API has the return value for activity zones incorrectly
+        typed as `int` when it can return either `int` or `float`.
+
         """
         zones = self.protocol.get("/activities/{id}/zones", id=activity_id)
+        # For troubleshooting
+        for z in zones:
+            print(z)
+            a = model.BaseActivityZone.model_validate(
+                {**z, **{"bound_client": self}}
+            )
+            print(a)
```

Comment:
```suggestion
```

Reply:
oh gosh - i meant to remove that it was just my testing things.